docs/install.rst

```diff
@@ -360,8 +360,9 @@ First install the build requirements with
 
 .. code-block:: bash
 
-   conda install sphinx
-   pip install sphinx_py3doc_enhanced_theme
+    conda install sphinx
+    pip install sphinx_py3doc_enhanced_theme
+    pip install sphinxcontrib-fulltoc
```

Comment:
just do a one-liner `pip install sphinx_py3doc_enhanced_theme sphinxcontrib-fulltoc`

Reply:
I knew you were going to raise this 🤣

Reply:
Are you testing my attention to detail? I predict it will drop dramatically after this week :smile: